tutorverse-client/src/components/ChatHome/ChatHome.jsx

```diff
@@ -1,49 +1,110 @@
 import { useNavigate } from "react-router-dom";
+import axios from "axios";
 import { useEffect, useState } from "react";
 import TextField from "@mui/material/TextField";
 import ArrowBackIosIcon from "@mui/icons-material/ArrowBackIos";
 import Button from "@mui/material/Button";
+import socketIO from "socket.io-client";
 import "./ChatHome.css";
 
-const messageList = [];
-export default function ChatHome({ socket, studentId, tutorId }) {
-  const [message, setMessage] = useState("");
+export default function ChatHome({ user }) {
+  const [tutors, setTutors] = useState([]);
+  const [messages, setMessages] = useState([]);
+  const [selectedUser, setSelectedUser] = useState(null);
+  const [newMessage, setNewMessage] = useState("");
+  const [socket, setSocket] = useState(null);
   const navigate = useNavigate();
 
+  //GET request for tutorlist
+  useEffect(() => {
+    const fetchTutors = async () => {
+      const response = await axios.get("http://localhost:3000/tutors", {
+        withCredentials: true,
+      });
+      setTutors(response.data.list);
+      console.log(response.data.list);
+    };
+    fetchTutors();
```

Comment:
I think we should move this logic to the server - here we should fetch a chat contacts API. Then on the server-side, return all tutors for a students, and then for a tutor, only show the the students that messaged them.